compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -557,6 +584,101 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
   }
 };
 
+/**
+ * @brief Quantize const input tensors using min/max of const values
+ */
+void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
+{
+  auto opcode = node->opcode();
+  auto arity = node->arity();
+
+  loco::Node *input_node{nullptr};
+  luci::CircleConst *const_node{nullptr};
+
+  switch (opcode)
+  {
+    case luci::CircleOpcode::CONV_2D:
+    case luci::CircleOpcode::DEPTHWISE_CONV_2D:
+    case luci::CircleOpcode::FULLY_CONNECTED:
+    case luci::CircleOpcode::TRANSPOSE_CONV:
+      // Handled in QuantizeWeights and QuantizeBias
+      break;
+
+    case luci::CircleOpcode::CONCATENATION:
+      // Handled in propagate_concat_quantparam
```

Comment:
Same question for the concatenation inputs.

Reply:
Since we need to check several conditions to propagate concat qparam, I think it is better to be handled separately.

Reply:
Current implementation of `propagate_concat_quantparam` does not quantize const inputs in some cases, e.g., if concat has a fused activation function, const inputs are not quantized now. To fix this, I'll make another PR after this PR is landed.

Reply:
Just thought that everyting in one place would be better :)